web/massastation/src/pages/Index/Index.tsx

```diff
@@ -30,7 +30,6 @@ export function Index() {
   const navigate = useNavigate();
   const [pluginWalletIsInstalled, setPluginWalletIsInstalled] = useState(false);
   const [urlPlugin, setUrlPlugin] = useState('');
-  const [theme]: string = useOutletContext();
```

Comment:
@mazmassa I have error _9() is not iterable on this line so I removed it, do you have any idea?

Reply:
This will disable the dark/light trigger. But no worries if fix your issues. I'll manage that later then in another way!